### PR TITLE
fix(security): vulnerabilities found in quorum-multi-party-all-in-one

### LINKED ISSUE #2060: fix(security): vulnerabilities found in quorum-multi-party-all-in-one
List of vulnerabilities found in quorum-multi-party-all-in-one image during Azure Container scan.
<html xmlns:v="urn:schemas-microsoft-com:vml"
xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:x="urn:schemas-microsoft-com:office:excel"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=Excel.Sheet>
<meta name=Generator content="Microsoft Excel 15">
<link id=Main-File rel=Main-File
href="file:///C:/Users/ZONDER~1.CAL/AppData/Local/Temp/msohtmlclip1/01/clip.htm">
<link rel=File-List
href="file:///C:/Users/ZONDER~1.CAL/AppData/Local/Temp/msohtmlclip1/01/clip_filelist.xml">

</head>

<body link="#0563C1" vlink="#954F72">

VULNERABILITY ID | PACKAGE NAME | SEVERITY
-- | -- | --
CVE-2021-36159 | apk-tools | CRITICAL
CVE-2021-30139 | apk-tools | HIGH
CVE-2022-28391 | busybox | CRITICAL
CVE-2021-28831 | busybox | HIGH
CVE-2021-42378 | busybox | HIGH
CVE-2021-42379 | busybox | HIGH
CVE-2021-42380 | busybox | HIGH
CVE-2021-42381 | busybox | HIGH
CVE-2021-42382 | busybox | HIGH
CVE-2021-42383 | busybox | HIGH
CVE-2021-42384 | busybox | HIGH
CVE-2021-42385 | busybox | HIGH
CVE-2021-42386 | busybox | HIGH
CVE-2021-36222 | krb5-libs | HIGH
CVE-2021-41617 | openssh-client | HIGH
CVE-2021-41617 | openssh-keygen | HIGH
CVE-2021-3711 | openssl | CRITICAL
CVE-2021-3450 | openssl | HIGH
CVE-2021-3712 | openssl | HIGH
CVE-2022-0778 | openssl | HIGH
CVE-2022-28391 | ssl_client | CRITICAL
CVE-2021-28831 | ssl_client | HIGH
CVE-2021-42378 | ssl_client | HIGH
CVE-2021-42379 | ssl_client | HIGH
CVE-2021-42380 | ssl_client | HIGH
CVE-2021-42381 | ssl_client | HIGH
CVE-2021-42382 | ssl_client | HIGH
CVE-2021-42383 | ssl_client | HIGH
CVE-2021-42384 | ssl_client | HIGH
CVE-2021-42385 | ssl_client | HIGH
CVE-2021-42386 | ssl_client | HIGH
CVE-2022-1271 | xz | HIGH
CVE-2022-1271 | xz-libs | HIGH
CVE-2018-25032 | zlib | HIGH
CVE-2021-32810 | crossbeam-deque | CRITICAL
CVE-2022-24713 | regex | HIGH
CVE-2022-24713 | regex | HIGH

</body>

</html>

### PR DESCRIPTION
**Pull request by aldousalvarez**
Fixes #2060

Signed-off-by: aldousalvarez <aldousss.alvarez@gmail.com>

### CODE REVIEW

**Comment by aldousalvarez** [comment]
Hello @petermetz , Most of the vulnerabilities are now fixed in quorum-multi-party-all-in-one but there are still some that are not because most of the remaining vulnerabilities are still new and is still waiting for the new changes to be pulled in and released on their respective package versions. And we tried to ask on the quorum discussions on github as you can see here. (https://github.com/ConsenSys/quorum/discussions/1513).

Here are the remaining vulnerabilities for quorum-multi-party-all-in-one:
CVE-2022-24713
CVE-2021-41092
CVE-2022-27664
CVE-2022-32149
CVE-2021-41092 
CVE-2022-32149

File: .github/containerscan/allowedlist.yaml [hunk 1]
```diff
@@ -1,6 +1,13 @@
...
 
-    -CVE-2022-37734
-    -CVE-2022-25857
+    #quorum-multi-party-all-in-one
+    - CVE-2022-24713
+    - CVE-2021-41092
+    - CVE-2022-27664
+    - CVE-2022-32149
+    - CVE-2021-41092
+    - CVE-2022-32149
```
Comment (petermetz) [inline .github/containerscan/allowedlist.yaml:13]: @aldousalvarez Same here as with the other quorum image PR: I would remove these from the ignored CVEs for now unless we have a strong reason to not to.

**Review by petermetz — CHANGES_REQUESTED** [review]

File: .github/containerscan/allowedlist.yaml (hunk 1 above)
Comment (aldousalvarez) [reply .github/containerscan/allowedlist.yaml]: Noted on that. Thank you, already fixed it

Comment (petermetz) [reply .github/containerscan/allowedlist.yaml]: @aldousalvarez Great, thanks for the updates!

**Review by petermetz — APPROVED** [review]
@aldousalvarez LGTM, thank you!

**Review by jagpreetsinghsasan — APPROVED** [review]
LGTM, thankyou!

**Comment by petermetz** [comment]
Removing my approval so that we don't accidentally merge before someone else also approves.

**Review by takeutak — APPROVED** [review]

**Review by petermetz — APPROVED** [review]
LGTM

**Pull request merged**